packages/application-shell/src/index.ts

```diff
@@ -8,7 +8,7 @@ export { applyDefaultMiddlewares } from './configure-store';
 export { default as InjectReducers } from './components/inject-reducers';
 export { default as RouteCatchAll } from './components/route-catch-all';
 export { default as setupGlobalErrorListener } from './utils/setup-global-error-listener';
-export { selectUserId, selectProjectKeyFromUrl } from './utils';
+export { selectUserId, selectProjectKeyFromUrl, getMcApiUrl } from './utils';
```

Comment:
A bit unfortunate but due to the location of the other helpers consistent.

Reply:
We don't need this export anymore